src/components/QuestionModal.jsx

```diff
@@ -17,11 +17,16 @@ const QuestionModal = props => {
     setShowingAnswer(!showingAnswer);
   };
 
-  const handleScore = () => {
+  const handleScoreCorrect = () => {
```

Comment:
Good job. But wouldn't it be better to have just one `handleScore` function that handles correct or wrong depending on a passed parameter?

I think it would, but feel free to debate with your reasoning.